src/StudyViewer/StudyDetails.jsx

```diff
@@ -403,7 +404,8 @@ class StudyDetails extends React.Component {
                    <div key={k}>
                      <div className='h3-typo'>{this.getLabel(k)}</div>
                      <Paragraph>
-                       {v}
+                       {/* strips out all html tags except br, strong, table, tr, td, th */}
+                       {parse(v.replace(/(<(?!(br\/?>)|(\/?strong>)|(\/?table>)|(\/?tr>)|(\/?td>|(\/?th>)))[^>]+>)/ig, ''))}
```

Comment:
do we need to consider any inline stylings exists inside tags? if so we may need to use something like `(\/?table.*>)`?

Reply:
```suggestion
                       {parse(v.replace(/(<(?!(br\/?>)|(\/?strong>)|(\/?table.*>)|(\/?tr>)|(\/?td>|(\/?th>)))[^>]+>)/ig, ''))}
```

Reply:
thanks added it